pkg/stub/handler.go

```diff
@@ -327,6 +337,75 @@ func (h *Handler) updateCfgArch(cfg *v1.Config) *v1.Config {
 	return cfg
 }
 
+func (h *Handler) imageConfigBlocksImageStreamCreation(name string) bool {
+	//TODO openshift/client-go/config/clientset/versioned/fake and ConfigV1Interface has compile issues with
+	// respect to ConfigV1Client (missing method implementations), so for now we cannot use it in our
+	// unit tests.  In the actual runtime if we cannot create the client the operator errors out on startup.
+	if h.configclient == nil {
+		return false
+	}
+	err := wait.PollImmediate(5*time.Second, 20*time.Second, func() (done bool, err error) {
+		// if the image config allowed registry or blocked registry list will prevent the creation of imagestreams,
+		// we consider this inaccessible
+		imgCfg, err := h.configclient.Images().Get(context.TODO(), "cluster", metav1.GetOptions{})
+		if err != nil {
+			logrus.Printf("unable to retrieve image configuration as part of testing registry.redhat.io connectivity: %s", err.Error())
+			return false, nil
```

Comment:
Would it make sense to just have this section in the `wait.PollImmediate` instead of the whole logic for this function?

Reply:
sorry not following / can't parse

Reply:
Would it make sense for only the Get() to be inside of the wait.PollImmediate, instead of all the code within the function being in there? Probably doesn't really matter honestly, just a thought.

Reply:
ah ... you are right that is better .... update up in a couple